CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+
+### 4. Governance NFT Types
+
+All governance NFTs proposed here share a **new CIP-67 label** (placeholder `(1694)`). Example asset names:
+
+- `"(1694)drepCredential-<dRepAlias>"`
+- `"(1694)ballotNote-<proposalId>"`
+- `"(1694)endorsement-<endorserKey>-<dRepId>"`
+
+#### 4.1 DRep Credential
+
+**Minted by:** The DRep or an authorized delegate.  
+**Purpose:**
+
+- Captures key DRep info on-chain (e.g., `dRepId`, name, roles).
+- Optionally anchors extended data via CIP-119.
+- Reflects the “DRep Credential Form” mockup fields.
+
+**Core Field:**
+
+- `dRepId` (CIP-1694 hash)
+
+**Recommended Optional Fields (Self-Reported):**
+
+- **Title/Role**: Short descriptor (e.g., “DeFi Developer”).
+- **Name (or Alias)**, **Profile Image URL**, **Website**, **Mission/Goals**.
+- **Primary Focus Areas** (e.g., DeFi, Governance, Education).
+- **Roles** (SPO, Dev/Builder, Community Manager, etc.).
+- **Sectors** (NFTs, Identity, Gaming, DAO Tools).
+- **Social Links** (Twitter handle, Discord handle).
+- **Off-Chain Anchor** (`cip119AnchorUrl`, `cip119AnchorHash`) for extended profiles.
+
+**Minimal JSON Example:**
+
+```jsonc
+{
+  "version": 1,
+  "dRepId": "b2f0...20fb",
+  "titleOrRole": "DeFi Developer",
+  "nameOrAlias": "Alice The Builder",
+  "missionOrGoals": "Promoting transparent DeFi governance on Cardano",
+  "primaryFocusAreas": ["DeFi", "Governance"],
+  "roles": ["Dev/Builder", "Community Manager"],
+  "website": "https://mydrepwebsite.com",
+  "twitterHandle": "@aliceBuilder",
+  "cip119AnchorUrl": "ipfs://bafy123...",
+  "cip119AnchorHash": "blake2b256:abc123..."
+}
+```
+
+#### 4.2 Ballot Note
+
+**Minted by:** A DRep publishing a stance on a proposal.  
+**Purpose:**
+
+- Declares the DRep’s intended vote (`Yes`, `No`, or `Abstain`) for a CIP-1694 proposal.
+- Can include a short rationale or reference CIP-108 anchor for deeper explanation.
+
+**Core Fields:**
+
+- `type = "ballotNote"`
+- `dRepId` (CIP-1694)
+- `proposalId` (CIP-1694)
+- `voteChoice`: `"Yes" | "No" | "Abstain"`
+
+**Optional Fields:**
+
+- `rationale`, `timestamp`
+- CIP-108 anchor (`anchorUrl`, `anchorHash`)
+
+**Minimal JSON Example:**
+
+```jsonc
+{
+  "version": 1,
+  "type": "ballotNote",
+  "dRepId": "b2f0...20fb",
+  "proposalId": "abcd1234+0",
+  "voteChoice": "Yes",
+  "rationale": "Supports Cardano's long-term growth",
+  "timestamp": "2025-03-10T12:00:00Z",
+  "anchorUrl": "ipfs://proposalDoc",
+  "anchorHash": "blake2b256:def456..."
+}
+```
+
+#### 4.3 Endorsement
+
+**Minted by:** Any stakeholder (SPO, DAO, community member) wishing to show support for a DRep.  
+**Purpose:**
+
+- On-chain “vote of confidence,” optionally with a comment or identity proof.
+- May include a **minimum ADA** amount to discourage spam.
+- Optionally allows endorsers to attach **additional** ADA to assist new DReps (e.g., raising the 500 ADA deposit) or provide financial backing.
```

Comment:
how can you endorse a DRep before theyre a DRep? how would the user know their payment address?

Reply:
Maybe with the ADA handles thing. Still working this out.